src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/LightCompiler.cs

```diff
@@ -2400,11 +2400,11 @@ private void CompileNewExpression(Expression expr)
 
                 if (updaters != null)
                 {
-                    _instructions.EmitByRefNew(node.Constructor, updaters.ToArray());
+                    _instructions.EmitByRefNew(node.Constructor, parameters, updaters.ToArray());
```

Comment:
Analogous to how we call `EmitCall` in the compiler, avoiding a second call to `GetParameters`.